documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -1,112 +1,267 @@
+transforming an S(**q**, w) at generation time.
 
 For example:
 
 ::
 
-   gen_sqw (spefile, par_file, sym_sqw_file, efix, emode, alatt, angdeg,...
-       u, v, psi, omega, dpsi, gl, gs,'transform_sqw',@(x)(symmetrise_sqw(x,v1,v2,v3)))
+   sym = SymopReflection(v1, v2, offset);
+   gen_sqw (spefile, par_file, sym_sqw_file, efix, emode, alatt, angdeg,...  u, v, psi, omega, dpsi, gl,
+            gs, 'transform_sqw', @(x)(symmetrise_sqw(x,sym)))
 
 
 or, more generally:
 
 ::
 
-   gen_sqw (spefile, par_file, sym_sqw_file, efix, emode, alatt, angdeg,...
-       u, v, psi, omega, dpsi, gl, gs,'transform_sqw',@(x)(user_symmetrisation_routine(x))
+   gen_sqw (spefile, par_file, sym_sqw_file, efix, emode, alatt, angdeg,...  u, v, psi, omega, dpsi, gl,
+            gs, 'transform_sqw', @user_symmetrisation_routine)
 
 
-where *spefile, par_file, etc...* are the options used during initial *sqw* file generation (see :ref:`Generating SQW files <manual/Generating_SQW_files:Generating SQW files>`). The first ``gen_sqw`` would build a *sqw* file reflected as in the example for the reflection in memory, but with the transformation applied to the entire dataset. In the second, more general, case the user defined function (in a m-file on the Matlab path) can define multiple symmetrisation operations that are applied sequentially to the entire data. An example is as follows, which folds a cubic system so that all eight of the symmetrically equivalent (1,0,0) type positions are folded on to each other:
+where ``spefile``, ``par_file``, etc... are the options used during initial *sqw* file generation (see :ref:`Generating
+SQW files <manual/Generating_SQW_files:Generating SQW files>`). The first ``gen_sqw`` would build a ``.sqw`` file
+reflected as in the example for the reflection above. In the second, more general, case the user defined function (in a
+``.m``-file on the Matlab path) can define multiple symmetrisation operations that are applied sequentially to the entire
+data. An example is as follows, which folds a cubic system so that all eight of the symmetrically equivalent (1,0,0)
+type positions are folded on to each other:
 
 ::
 
    function wout = user_symmetrisation_routine(win)
 
-   wout=symmetrise_sqw(win,[1,1,0],[0,0,1],[0,0,0]);%fold about line (1,1,0) in HK plane
-   wout=symmetrise_sqw(wout,[-1,1,0],[0,0,1],[0,0,0]);%fold about line (-1,1,0) in HK plane
-   wout=symmetrise_sqw(wout,[1,0,1],[0,1,0],[0,0,0]);%fold about line (1,0,1) in HL plane
-   wout=symmetrise_sqw(wout,[1,0,-1],[0,1,0],[0,0,0]);%fold about line (1,0,-1) in HL plane
+   wout=symmetrise_sqw(win, SymopReflection([1,1,0], [0,0,1]));%fold about line (1,1,0) in HK plane
+   wout=symmetrise_sqw(wout,SymopReflection([-1,1,0],[0,0,1]));%fold about line (-1,1,0) in HK plane
+   wout=symmetrise_sqw(wout,SymopReflection([1,0,1], [0,1,0]));%fold about line (1,0,1) in HL plane
+   wout=symmetrise_sqw(wout,SymopReflection([1,0,-1],[0,1,0]));%fold about line (1,0,-1) in HL plane
```

Comment:
- missing end on the end
- can the symopreflections not be combined in an array and put into a single call to symmetrise_sqw? If not, should probably be commented on